src/components/CommentList/index.ts

```diff
@@ -1 +1,2 @@
+// eslint-disable-next-line
```

Comment:
Disabling eslint for a single line should be avoided if possible. Try to fix the issue causing the eslint warning or error instead of disabling it.

Reply:
something was wrong in develop process - removed now - thank you for highlighting that point